src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -13733,6 +13733,99 @@ def test_aks_create_with_enable_acns_complex(
             checks=[self.is_empty()],
         )
 
+    @AllowLargeResponse()
```

Comment:
you can mark the case as `@live_only()` as it's testing the monitor addon, which current recording mechanism is not suitable